api/proto/common/common.proto

```diff
@@ -8,3 +8,12 @@ message G1Commitment {
 	// The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
 	bytes y = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////
+// Messages for WIP RPCs. These are subject to change, do not consider these to be stable API. //
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+// A chunk of a blob.
+message ChunkData {
+	bytes data = 1;
```

Comment:
Users need a way to decode the bytes. We may settle on Gnark, so maybe it's fine to just have bytes and implicitly assume it's Gnark.

Reply:
My thinking here for just having bytes is that once we settle on a single encoding, we can document the format. If we end up deciding to support multiple encodings, we can trivially add another field to specify the encoding.